git-rewind-rebase: Make it the original file

guludo opened the pull request:
The term rewind-rebase is arguably more descriptive to what the command does than recursive-rebase (there is no real recursion). As such, let's make git-rewind-rebase the main file and have git-recursive-rebase be the symbolic link.

This is relevant if, for example, we want to point to the command with a GitHub URL. With the current state, [1] would only show "git-recursive-rebase" as the content of the symlink.

[1] https://github.com/lucasdemarchi/toolbox/blob/master/git-rewind-rebase

guludo committed:
git-rewind-rebase: Make it the original file

The term rewind-rebase is arguably more descriptive to what the command
does than recursive-rebase (there is no real recursion). As such, let's
make git-rewind-rebase the main file and have git-recursive-rebase be
the symbolic link.

This is relevant if, for example, we want to point to the command with a
GitHub URL. With the current state, [1] would only show
"git-recursive-rebase" as the content of the symlink.

[1] https://github.com/lucasdemarchi/toolbox/blob/master/git-rewind-rebase

Signed-off-by: Gustavo Sousa <gustavo.sousa@intel.com>

lucasdemarchi commented:
There some occurrences of "recursive rebase" in the script. Maybe add a commit on top renaming those and rewording the comments?. Thanks. LGTM

guludo committed:
git-rewind-rebase: Reword to align with "rewind"

Now that git-rewind-rebase is the new official name, reword the script
to better align with that.

Signed-off-by: Gustavo Sousa <gustavo.sousa@intel.com>

guludo commented:
> There some occurrences of "recursive rebase" in the script. Maybe add a commit on top renaming those and rewording the comments?. Thanks. LGTM

Done.

Even though you have already approved, I made some substantial changes in the script's description, so I think a new ack is welcome :-)
